shell/platform/darwin/macos/framework/Source/FlutterEngine_Internal.h

```diff
@@ -17,10 +18,16 @@
 @property(nonatomic, readonly) BOOL running;
 
 /**
- * Provides the renderer config needed to initialize the engine and also handles external texture
- * management.
+ * Provides the openGL renderer config needed to initialize the engine and also handles external
+ * texture management.
  */
-@property(nonatomic, readonly, nonnull) FlutterOpenGLRenderer* openGLRenderer;
+@property(nonatomic, readonly, nullable) FlutterOpenGLRenderer* openGLRenderer;
+
+/**
+ * Provides the metal renderer config needed to initialize the engine. When using metal rendering
+ * external textures are not supported.
+ */
+@property(nonatomic, readonly, nullable) FlutterMetalRenderer* metalRenderer;
```

Comment:
Similar to my comment in the other file, I'm not sure why there are two renderer ivars.